1714897138015	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1714897211388	Sync.SyncScheduler	DEBUG	Going to sleep, doing a quick sync.
1714897211388	Sync.Service	DEBUG	User-Agent: Firefox/125.0.3 (Linux x86_64) FxSync/1.127.0.20240503195641.desktop
1714897211388	Sync.Service	INFO	Starting sync at 2024-05-05 09:20:11 in browser session xh_alQ_XLh5n
1714897211389	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1714897211394	Sync.Status	INFO	Resetting Status.
1714897211394	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1714897211394	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1714942555405	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1714942555405	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1714942555408	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1714942555409	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1714942555590	Sync.SyncScheduler	DEBUG	Woke from sleep.
1714942555590	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1714942555590	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1714942555591	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1714942557592	Sync.Service	DEBUG	User-Agent: Firefox/125.0.3 (Linux x86_64) FxSync/1.127.0.20240503195641.desktop
1714942557593	FirefoxAccounts	INFO	Polling device commands.
1714942557593	Sync.Service	INFO	Starting sync at 2024-05-05 21:55:57 in browser session xh_alQ_XLh5n
1714942557593	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.sys.mjs:43:5) JS Stack trace: LockException@util.sys.mjs:43:5
throwLockException@util.sys.mjs:127:11
WrappedLock@util.sys.mjs:141:15
_lockedSync@service.sys.mjs:1372:6
sync/<@service.sys.mjs:1337:18
WrappedCatch@util.sys.mjs:112:27
sync@service.sys.mjs:1338:7
1714942557593	Sync.Service	INFO	Cannot start sync: already syncing?
1714942557594	Services.Common.RESTRequest	DEBUG	GET request to https://api.accounts.firefox.com/v1/account/device/commands?index=0
1714942557596	Services.Common.RESTRequest	DEBUG	GET https://api.accounts.firefox.com/v1/account/device/commands?index=0 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1714942557596	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.sys.mjs:478
1714942557596	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
request()@resource://services-common/hawkclient.sys.mjs:230
1714942557597	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 630"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:630
request()@resource://services-common/hawkclient.sys.mjs:238
1714942557597	Hawk	DEBUG	(Response) /account/device/commands?index=0: code: null - Status text: null
1714942557597	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:246
1714942557597	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:247
1714942557597	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 630"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:630
_constructError()@resource://services-common/hawkclient.sys.mjs:113
request()@resource://services-common/hawkclient.sys.mjs:252
1714942557597	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
_constructError()@resource://services-common/hawkclient.sys.mjs:114
request()@resource://services-common/hawkclient.sys.mjs:252
1714942557597	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
_constructError()@resource://services-common/hawkclient.sys.mjs:115
request()@resource://services-common/hawkclient.sys.mjs:252
1714942557597	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_constructError()@resource://services-common/hawkclient.sys.mjs:121
request()@resource://services-common/hawkclient.sys.mjs:252
1714942557597	FirefoxAccounts	ERROR	error GETing /account/device/commands?index=0: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]","message":null,"code":null,"errno":null}
1714942557597	Sync.SyncScheduler	ERROR	Fetching missed remote commands failed.: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]","message":null,"code":null,"errno":null}
1714942559475	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1714942559475	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1714942559477	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1714942559478	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1714942559479	Sync.Service	DEBUG	User-Agent: Firefox/125.0.3 (Linux x86_64) FxSync/1.127.0.20240503195641.desktop
1714942559479	Sync.Service	INFO	Starting sync at 2024-05-05 21:55:59 in browser session xh_alQ_XLh5n
1714942559479	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.sys.mjs:43:5) JS Stack trace: LockException@util.sys.mjs:43:5
throwLockException@util.sys.mjs:127:11
WrappedLock@util.sys.mjs:141:15
_lockedSync@service.sys.mjs:1372:6
sync/<@service.sys.mjs:1337:18
WrappedCatch@util.sys.mjs:112:27
sync@service.sys.mjs:1338:7
1714942559479	Sync.Service	INFO	Cannot start sync: already syncing?
1714942561041	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1714942562024	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/204028242/info/collections failed: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1714942562025	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1714942562025	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1714942562027	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
